JM/CF/Scripts/3_Game/CommunityFramework/TypeConverter/CF_TypeConverterDate.c

```diff
@@ -0,0 +1,41 @@
+class CF_TypeConverterDate : CF_TypeConverterManaged
+{
+	private string m_Format = "YYYY-MM-DD hh:mm:ss";
```

Comment:
Fixed format? Ehhh ... I guess no way around it?

Reply:
For this PR, yes.

In the future, `CF_Date` may store its format but that is currently undecided.